implementingSwashbuckle.md

```diff
@@ -0,0 +1,133 @@
+             schemaFilters: new List<ISchemaFilter>() {
+                 new ApplySwaggerSchemaFilterAttributes()
+                 },
+             operationFilters: new List<IOperationFilter>() {
+              //Recognize XML comments in the code and write them to the MyAPI.XML file.
+                 new ApplyXmlActionComments("MyAPI.XML"),
+              //Enable response code classes and attributes through Swashbuckle.
+                 new ApplySwaggerResponseAttributes(),
+              //Modify operation descriptions after they are generated.
+                 new ApplySwaggerOperationAttributes(),
+              //Enable operation filters and attributes.
+                 new ApplySwaggerOperationFilterAttributes()
+             }
+
+         )
+
+     );
+     var swaggerDoc = swaggerProvider.GetSwagger("http://localhost:8006", "v1");
+
+  // Serializes the swaggerDoc variable.
+     var swaggerString = JsonConvert.SerializeObject(
+         swaggerDoc,
+         Formatting.Indented,
+         new JsonSerializerSettings
+         {
+             NullValueHandling = NullValueHandling.Ignore,
+             Converters = new[] { new VendorExtensionsConverter() }
+         }
+     );
+
+```
+## Using DapperDox
+
+[DapperDox](http://dapperdox.io/) is an open source documentation framework for OpenAPI specifications. Instead of having Swashbuckle publish our API specificaiton in the Swagger UI, I added the following code to the ```Startup.cs``` file. This code writes the Swagger specification to a swagger.json file.
+
+```
+   System.IO.StreamWriter file = new System.IO.StreamWriter("swagger.json");
+      file.WriteLine(swaggerString);
+      file.Close();
+```
+
+DapperDox reads this file and displays it in its own UI. I installed DapperDox and pointed it at my swagger.json file, and saw nothing but error messages in my command prompt.
+
+Reading through the [DapperDox documentation](http://dapperdox.io/docs/spec-resource-definitions), I discovered that "When specifying a resource schema object...DapperDox requires that the optional schema object title member is present." This requirement was problematic, because Swashbuckle does not include a method for adding members to a schema in the generated swagger.json file. Additionally, it took some tinkering in the code for me to realize that the missing title member on the ```definitions``` model is what caused DapperDox to break.
+
+## Fixing the output
+
+The Swashbuckle documentation offered little help in this regard, so I turned to one of our developers. After reviewing the code together, my developer counterpart created a ```SwaggerTitleFilter``` method that adds a title member to the definitions model in the resulting swagger.json file. The title member displays in the generated documentation as a link to the referenced object, creating a hyperlink between the two objects.
+
+The following code implements an IModelFilter that causes Swashbuckle to generate a title member for any schema.
```

Comment:
Could also markup `IModelFilter` inline to make it stand out. This is a nice example, and shows what you end up having to do to get all the display you want. Nice.